src/components/status-card/status-card.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright contributors to the ibm-storage-odf-console project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Gallery,
+  GalleryItem,
+} from "@patternfly/react-core";
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import {
+  HealthItem,
+} from "@console/dynamic-plugin-sdk/provisional";
+import { getFlashsystemHealthState } from './utils';
+import { StorageInstanceKind } from '../../types';
+//import { StorageInstanceModel } from '../../../../models';
+import {GetFlashSystemResource} from '../../constants/resources'
+
+export const StatusCard: React.FC<any> = (props) => {
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
+  const flashHealthState = getFlashsystemHealthState({ sto: { data: data, loaded: loaded, loadError: loadError } });
+
+  return (
+    <Card className="co-dashboard-card co-dashboard-card--gradient">
+      <CardHeader className="co-dashboard-card__header">
+        <CardTitle className="co-dashboard-card__title">Status</CardTitle>
+      </CardHeader>
+      <CardBody className="co-dashboard-card__body">
+        <div className="co-dashboard-card__body--top-margin co-status-card__health-body">
+          <Gallery className="co-overview-status__health" hasGutter>
```

Comment:
@bipuladh all these css will be loaded by default for all packages ?

Reply:
updated with latest dashborad components.